packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -57,11 +59,16 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
         (window as any).Fabric = Fabric;
       });
     }
+
+    if (props.isCodeVisible !== undefined && props.onToggleEditor === undefined && process.env.NODE_ENV !== 'production') {
+      warn('erro: Missing onToggleEditor prop.');
```

Comment:
Probably should be a little more descriptive. Something like `ExampleCard: The onToggleCodeEditor prop is required if isCodeVisible is provided. Otherwise the show/hide code button will not work.`